modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionCommandListener.java

```diff
@@ -163,4 +308,26 @@ else if (clo.command() instanceof UpsertCommand) {
             return hash;
         }
     }
+
+    /**
+     * @param row Row.
+     * @return Extracted key.
+     */
+    @NotNull private boolean equalValues(@NotNull BinaryRow row, @NotNull BinaryRow row2) {
```

Comment:
you don't need @NotNull for return values as far as the return value is primitive.

Reply:
Done.